native/cocos/renderer/pipeline/RenderPipeline.h

```diff
@@ -142,6 +147,7 @@ class CC_DLL RenderPipeline : public Asset {
     PipelineUBO *_pipelineUBO{nullptr};
     scene::Model *_profiler{nullptr};
     IntrusivePtr<PipelineSceneData> _pipelineSceneData;
+    GeometryRenderer *_geometryRenderer{nullptr};
```

Comment:
So, it is a weak reference?

Reply:
Yes, it's assigned from a raw pointer type